R/teal.data.R

```diff
@@ -13,6 +13,6 @@
 #' @import shiny
 #' @importFrom digest digest
 #' @importFrom stats setNames
-#' @importFrom shinyjs show
+#' @importFrom shinyjs useShinyjs
```

Comment:
why this?

Reply:
this is why https://github.com/daattali/shinyjs/blob/67b269df8f07860c6be8dd10fb90575eee840c8d/NAMESPACE#L31

`shinyjs` exports method show which mask generic method! But don't worry, `shinyjs` dependency will be removed soon after internal release.